pages/express-relay/integrate-as-searcher.mdx

```diff
@@ -0,0 +1,258 @@
+import { Callout, Tabs, Steps } from "nextra/components";
+
+# How to Integrate Express Relay as a Searcher
+
+Express Relay gives one place stop for on-chain opportunities. Searchers can access opportunities from various DeFi protocols via a simple and unified interface.
```

Comment:
For searchers, I think we are providing API not interface. So maybe instead of `simple and unified interface`, we can use `simple and unified API`?

Reply:
We already removed it in latest revision.